docs/readthedocs/source/doc/LLM/Overview/install_gpu.md

```diff
@@ -397,7 +466,7 @@ To use GPU acceleration on Linux, several environment variables are required or
 
       .. code-block:: bash
 
-         # Required step. Configure oneAPI environment variables
```

Comment:
Skip this step for pip-installed oneAPI since it has already been configured in LD_LIBRARY_PATH?